src/components/diagrams/singleLineDiagram/single-line-diagram-content.tsx

```diff
@@ -434,7 +488,7 @@ function SingleLineDiagramContent(props) {
                 !computationStarting && !isAnyNodeBuilding && !modificationInProgress && !props.loadingState;
 
             const diagramViewer = new SingleLineDiagramViewer(
-                svgRef.current, //container
+                svgRef.current!, //container
```

Comment:
i think the ref will always be initilize in that particular case :
```
 <Box
                ref={svgRef}
                sx={mergeSx(
                    styles.divDiagram,

```

then SingleLineDiagramViewer is created in useLayoutEffect.

Reply:
If I understand correctly, it will always be defined, so it is safer to add a test in the if statement 5 lines above. It won't change anything from your point of view, but you won't need the force operator.